### ISSUE TITLE
Remove pylama

### ISSUE THREAD

**Issue by amyheather** [lintquarto/lintquarto#25]
Pylama aggregates linters. It would make sense to allow users to specify which linters to run with pylama. Pylama uses an -l flag to define them: `pylama -l "pycodestyle,mccabe"`

From pylama docs:

```
  --linters LINTERS, -l LINTERS
                        Select linters. (comma-separated). Choices are eradicate,mccabe,mypy,pycodestyle,pydocstyle,pyflakes,pylint,isort.
```

**Comment by amyheather** [comment]
Actually decided to remove pylama, as - of its linter options:

* mypy, pycodestyle, pyflakes, pylint - already supported
* pydocstyle - ruled out as deprecated (#12)
* mccabe - ruled out as similarity to flake8 (#12)
* eradicate - ruled out as multi-line output (#12)
* isort - ruling out as it works on quarto files natively

Hence, have realised it doesn't super make sense to include! But this does raise the point that being able to run multiple at one would be great so opened a new issue...
